src/axom/sidre/core/View.cpp

```diff
@@ -922,6 +1029,37 @@ void View::createNativeLayout(Node& n) const
   n.set_external(m_node.schema(), data_ptr);
 }
 
+/*
+ *************************************************************************
+ *
+ * Deep-copy data to given Conduit node.
+ *
+ *************************************************************************
+ */
+void View::deepCopyToConduit(Node& dst) const
+{
+  // see ATK-726 - Handle undescribed and unallocated views in Sidre's
+  // createNativeLayout()
+  // TODO: Need to handle cases where the view is not described
+  // TODO: Need to handle cases where the view is not allocated
+  // TODO: Need to handle cases where the view is not applied
+
+  const conduit::DataType& srcDtype = m_node.dtype();
+  dst.set(srcDtype);
+  if(isAllocated())
+  {
+    // Using set_node to set dst: would reset dst's allocator id to
+    // the Conduit default (not what we want) if dst is an object or
+    // list.  Fortunately, Sidre never uses a Conduit node in those
+    // modes.
```

Comment:
Is there a reason/guarantee that this will not change? Should there be a check for this?

Reply:
I don't see this changing, but I'm not 100% sure.  If we Node as an object or a list, `Group` would own a node.  Only `View` owns a node.

Reply:
I'm pretty sure we have some list views -- @nselliott knows the details